arwen/elrondapi/manBufOps.go

```diff
@@ -380,6 +416,7 @@ func v1_4_mBufferAppendBytes(context unsafe.Pointer, accumulatorHandle int32, da
 
 	gasToUse = math.MulUint64(metering.GasSchedule().BaseOperationCost.DataCopyPerByte, uint64(len(data)))
 	metering.UseGas(gasToUse)
+	runtime.TraceGasUsed(mBufferAppendBytesName, initialGasLeft-metering.GasLeft())
```

Comment:
it would be better to have something like AddToTracedGas to simplify the actions.